internal/manager/run.go

```diff
@@ -171,6 +173,21 @@ func Run(ctx context.Context, c *Config, diagnostic util.ConfigDumpDiagnostic, d
 		return fmt.Errorf("unable to setup readyz: %w", err)
 	}
 
+	if c.Konnect.ConfigSynchronizationEnabled {
+		setupLog.Info("Start Konnect client to register runtime instances to Konnect")
+		konnectClient, err := konnect.NewAdminClient(c.Konnect)
+		if err != nil {
+			setupLog.Error(err, "failed to create konnect client")
+			return fmt.Errorf("failed to create konnect client: %w", err)
```

Comment:
This will cause this log to be printed twice, no?

Reply:
updated.